optimum/exporters/neuron/convert.py

```diff
@@ -201,6 +206,8 @@ def export_neuronx(
 
     dummy_inputs = config.generate_dummy_inputs(**input_shapes)
     logger.info(f"Using Neuron: --auto-cast {auto_cast}")
+
+    auto_cast = auto_cast + "t" if auto_cast == "matmul" else auto_cast
```

Comment:
```suggestion
    auto_cast = "matmult" if auto_cast == "matmul" else auto_cast
```
Let's make thing explicit!